### PR TITLE
Textarea

### PR DESCRIPTION
**Pull request by vinyll**

### CODE REVIEW

File: styleguide.css [hunk 1]
```diff
@@ -227,17 +226,22 @@ li a {
...
     display: block;
     width: 30rem;
+    transition: border-color .3s;
 }
-input:focus {
+input:focus,
+textarea:focus {
     border: 0.1rem solid #bde513;
     outline: none;
+    transition: border-color .3s;
```
Comment (vinyll) [inline styleguide.css:244]: you probably want this transition on the normal state, not on `:focus` (like you did for `input` and `button`)

File: styleguide.html [hunk 2]
```diff
@@ -63,4 +62,6 @@
     <input type="checkbox" id="styleguide" />
     <label for="styleguide">Checkbox</label>
   </p>
-</form>
\ No newline at end of file
+</form>
+
+<textarea></textarea>
```
Comment (vinyll) [inline styleguide.html:67]: you may eventually need to move this into `<form>` to tune forms and fields more realistically.

**Review by vinyll — COMMENTED** [review]
Not bad at all 👍

File: styleguide.html (hunk 2 above)
Comment (johangiraud) [reply styleguide.html:67]: ok

File: styleguide.css (hunk 1 above)
Comment (johangiraud) [reply styleguide.css:244]: ok

**Comment by vinyll** [comment]
Do you have more questions on the updates here @cafebabel-webmaster?

**Pull request merged**